test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -47,9 +47,10 @@ internal static ProjectDirectory Project
 #endif
         }
 
-        internal Task<MSBuildResult> DotnetMSBuild(string target)
+        internal Task<MSBuildResult> DotnetMSBuild(string target, string args = null, bool debug = false)
         {
-            return MSBuildProcessManager.RunProcessAsync($"/t:{target}", Project.DirectoryPath);
+            var timeout = debug ? (TimeSpan?)Timeout.InfiniteTimeSpan : null;
+            return MSBuildProcessManager.RunProcessAsync($"/t:{target} {args}", Project.DirectoryPath, timeout);
```

Comment:
We need this or else MSBuild gets killed. I wanted to separate it from `Debugger.IsAttached` so I can use it at the command line.